samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -5,28 +5,30 @@
  */
 
 &mram1x {
-	/delete-node/ cpuapp_rw_partitions;
+	/delete-node/ cpuapp-rw-partitions;
 
 	cpuapp_rw_partitions: cpuapp-rw-partitions {
 		compatible = "nordic,owned-partitions", "fixed-partitions";
 		status = "okay";
-		perm-read;
-		perm-write;
-		perm-secure;
+		nordic,access = <NRF_OWNER_ID_APPLICATION NRF_PERM_RWS>;
 		#address-cells = <1>;
 		#size-cells = <1>;
 
 		dfu_partition: partition@100000 {
 			reg = <0x100000 DT_SIZE_K(908)>;
 		};
 
-		storage_partition: partition@1e3000 {
-			reg = <0x1e3000 DT_SIZE_K(20)>;
+		/* Align the partition size to 4096 B to avoid gaps. */
+		bt_fast_pair_partition: partition@1e3000 {
+			reg = <0x1e3000 DT_SIZE_K(4)>;
 		};
 
-		bt_fast_pair_partition: partition@1e8fb8 {
-			label = "bt_fast_pair";
-			reg = <0x1e8fb8 0x48>;
+		/* The storage partition needs to start at address divisible
```

Comment:
@alstrzebonski I don't think you need here to justify why the address starts at 0x1e4000 as this is where the previous partition stops.
+ The justification is misleading. Currently the storage system doesn't require an alignment of the start address to the sector size. There might be a bug there that needs to be fixed later.

Reply:
I don't agree. There's a bug here so currently the storage system does require an alignment. If the bug is fixed then it will no longer require it, but for now it does.

Reply:
Much simpler : remove the comment in the device tree, but keep it in the commit message.
Why do you need to justify in the device tree that the address starts at 4K boundary if the previous one stops there ?

Reply:
Ok, I will remove the comment. Will you take over the bug or do I need to open issue in Zephyr?

Reply:
@alstrzebonski I will take over the bug